flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/plan/TimeIndicatorConversionTest.scala

```diff
@@ -253,9 +253,9 @@ class TimeIndicatorConversionTest extends TableTestBase {
         ),
         term("groupBy", "long"),
         term("window", "TumblingGroupWindow('w2, 'newrowtime, 1000.millis)"),
-        term("select", "long", "SUM(int) AS TMP_3", "end('w2) AS TMP_2")
+        term("select", "long", "SUM(int) AS TMP_1", "end('w2) AS TMP_0")
```

Comment:
side comment: If we would `TMP_` to `EXPR$`, I think we would get closer to Calcite's output.